test/PerformanceTests/SerializationComparisonsTests/Lib/DefaultWriterCollection.cs

```diff
@@ -0,0 +1,62 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DefaultWriterCollection.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Text.Json;
+using Microsoft.OData.Edm;
+
+namespace ExperimentsLib
+{
+    /// <summary>
+    /// Creates the default collection of <see cref="IPayloadWriter{T}"/>s used for testing.

```

Comment:
nit: is this `s` necessary here?

Reply:
I assume you mean the one after `<see cref"IPayloadWriter{T}"/>`s? It just makes it plural, I don't think it's an issue.

But I've replaced it with `<see cref="IPayloadWriter{T}"/> instances`.